bob/src/api/mod.rs

```diff
@@ -366,12 +366,22 @@ async fn find_group<A: Authenticator>(
         .iter()
         .find(|dc| dc.vdisks().iter().any(|&vd| vd == vdisk_id))
         .ok_or_else(|| {
-            let err = format!("Disk Controller with vdisk #{} not found", vdisk_id);
+            let dcs = dcs.iter()
+                .map(|dc| format!("DC: {:?}, vdisks: {}",
+                                  dc.disk(),
```

Comment:
It is better to print only the name
```suggestion
                .map(|dc| format!("DC: {}, vdisks: {}",
                                  dc.disk().name(),
```